sdk/python/kfp/components/executor_test.py

```diff
@@ -483,59 +615,63 @@ def test_artifact_output(self):
     }
     """
 
-        def test_func(first: str, second: str) -> Artifact:
+        def test_func(first: str, second: str, output: Output[Artifact]) -> str:
```

Comment:
I know at some point I mentioned that output artifact via return won't work. I was wrong. We should keep such test.

Reply:
That being said, after your change, the executor input doesn't match `test_func` signature as there's no output parameter. Interestingly this mismatch doesn't produce any error?

Reply:
Keep a test case with `-> Artifact`? (likely don't need to change this test func at all)